cli/module_build.go

```diff
@@ -55,7 +56,7 @@ func ModuleBuildStartAction(cCtx *cli.Context) error {
 }
 
 func (c *viamClient) moduleBuildStartAction(cCtx *cli.Context) error {
-	manifest, err := loadManifest(cCtx.String(moduleFlagPath))
+	manifest, err := loadManifest(cCtx.String(moduleFlagPath), false)
```

Comment:
[aside] Could be cool to have a `--dev` flag to the build cmd to change this behavior.

Reply:
(we should probably leave that to a followup pr)

Reply:
yup followup